tests/rust/test_rust_backend.py

```
@@ -353,7 +354,7 @@
             for cfg in invalid_configs:
                 try:
                     rust_backend.set_config(cfg)
-                except (ValueError, TypeError, Exception) as e:
+                except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)